src/components/FeeSelector/shared.tsx

```diff
@@ -1,6 +1,7 @@
 import { Trans } from '@lingui/macro'
 import { FeeAmount } from '@uniswap/v3-sdk'
-import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from 'constants/chains'
+import { SupportedChainId } from 'constants/chains'
```

Comment:
surprised your linter didn't consolidate to one line

Reply:
i did this using find and replace in shell